apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -64,6 +65,13 @@ export default function ListBox({
   const { setStoreValue } = useDataStore(model);
   const loadMoreItems = useCallback(itemsLoader.loadMoreItems, [layout]);
 
+  const overflowDisclaimerReducer = (state, action) => ({ ...state, ...action });
+  const [overflowDisclaimer, setOverflowDisclaimer] = useReducer(overflowDisclaimerReducer, {
+    show: false,
+    dismissed: false,
+  });
+  const dismissOverflowDisclaimer = () => setOverflowDisclaimer({ dismissed: true });
```

Comment:
Instead of a reducer (which by convention should *dispatch* an action `type` instead of a setter) you could simply use a useState with a function:
```
const [overflowDisclaimer, setOverflowDisclaimer] = useState({
    show: false,
    dismissed: false,
});
const dismissOverflowDisclaimer = () => setOverflowDisclaimer((state) => ({ ...state, dismissed: true }));
```

Otherwise, I think using reducers is a nice idea for more complex state management. I saw this guide: https://beta.reactjs.org/learn/extracting-state-logic-into-a-reducer